samples/bot-message-reminder/csharp/README.md

```diff
@@ -0,0 +1,124 @@
+﻿﻿﻿
+## Message reminder with messaging extension action
+
+This sample shows a feature where user can schedule a task from messaging extension action and get a reminder card at a scheduled time.
+
+![Select message](MessagingExtensionReminder/Images/MessageAction.png)
+
+![Task Details](MessagingExtensionReminder/Images/ScheduleTask.png)
+
+![Task reminder](MessagingExtensionReminder/Images/TaskReminder.png)
+
+![Team message action ](MessagingExtensionReminder/Images/MessageActionTeam.png)
+
+![Team Task Details ](MessagingExtensionReminder/Images/ScheduleTaskTeam.png)
+
+![ TeamTask reminder](MessagingExtensionReminder/Images/TaskReminderTeam.png)
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  determine dotnet version
+  ```bash
+  dotnet --version
+  ```
+- [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
+  
+  run ngrok locally
+  ```bash
+  ngrok http -host-header=localhost 3978
+  ```
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Setup for Bot
+   In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
+   
+2) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+   
+3) In folder where repository is cloned, navigate to `samples/bot-message-reminder/csharp`
+
+    change into project folder
+    ```bash
+    cd # MessagingExtensionReminder
+    ```
+    
+4) Run ngrok - point to port 3978
+
+    ```bash
+    # ngrok http -host-header=rewrite 3978
+    ```
+ 
+5) Modify the `manifest.json` in the `/AppPackage` folder and replace the following details:
+  - `{{Microsoft-App-Id}}` with Application id generated from Step 1
+  - `{{domain-name}}` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok.io` then your domain-name will be *.ngrok.io.
+
+6) Zip the contents of `AppPackage` folder into a `manifest.zip`, and use the `manifest.zip` to deploy in app store or add to Teams using step 9.
+
+7) Modify the `/appsettings.json` and fill in the following details:
```

Comment:
Without asking user to open the project solution we are asking user to update appsettings.json which is very confusing.
Please add those steps.